deps/Makefile

```diff
@@ -161,6 +161,15 @@ libconfig/libconfig/lib/.libs/libconfig++.a:
 
 libconfig: libconfig/libconfig/lib/.libs/libconfig++.a
 
+prometheus-cpp/lib/libprometheus-cpp-core.a:
+	cd prometheus-cpp && tar --strip-components=1 -zxf v0.9.0.tar.gz -C .
+	cd prometheus-cpp && tar --strip-components=1 -zxf civetweb-v1.11.tar.gz -C 3rdparty/civetweb
+	cd prometheus-cpp && tar --strip-components=1 -zxf googletest-v1.10.0.tar.gz -C 3rdparty/googletest
+	cd prometheus-cpp && patch -p1 < patches/serial_exposer.patch
```

Comment:
A sub-directory to store the patch for the parent directory: this goes against the convention of all other patches we have.
The patch should be in the parent directory, and applied to a sub-directory.

Reply:
Patch is now in parent directory. Done!